articles/openshift/configure-azure-ad-cli.md

```diff
@@ -17,47 +17,67 @@ If you choose to install and use the CLI locally, this article requires that you
 
 Retrieve your cluster-specific URLs that are going to be used to configure the Azure Active Directory application.
 
-Construct the cluster's OAuth callback URL and store it in a variable **oauthCallbackURL**. Make sure to replace **aro-rg** with your resource group's name and **aro-cluster** with your cluster's name.
+Set the variables for resource group and cluster name.
+
+Replace **\<resource_group>** with your resource group's name and **\<aro_cluster>** with your cluster's name.
+
+```azurecli-interactive
+resource_groupg=<resource_group>
```

Comment:
There is an extra 'g' at the end of `resource_groupg. Do you mind updating this?

Reply:
Fixed.